CHANGELOG.md

```diff
@@ -12,6 +12,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Before this change, `ember-apollo-client` would always wrap the returned data
   from the GraphQL (when not an array) with `EmberObject`. In this PR we removed
   that and instead we return plain objects.
+
+- Remove usage of Evented mixin: `Subscribe` no longer returns and Ember Object with Evented. Instead, it just
```

Comment:
```suggestion
- Remove usage of Evented mixin: `subscribe` no longer returns and Ember Object with Evented. Instead, it just
```
You may also want to state more explicitly, that not only the `Evented` Mixin is dropped, but also the `EmberObject` base class, meaning that `subscription.get('lastEvent')` would have to be replace with `get(subscription, 'lastEvent')` or even better yet `subscription.lastEvent`, if supported by the Ember version.